packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -229,22 +259,29 @@ class CupertinoDialogAction extends StatelessWidget {
   Widget build(BuildContext context) {
     TextStyle style = _kCupertinoDialogActionStyle;
 
-    if (isDefaultAction)
+    if (isDefaultAction) {
       style = style.copyWith(fontWeight: FontWeight.w600);
+    }
 
-    if (isDestructiveAction)
+    if (isDestructiveAction) {
       style = style.copyWith(color: CupertinoColors.destructiveRed);
+    }
 
-    if (!enabled)
+    if (!enabled) {
       style = style.copyWith(color: style.color.withOpacity(0.5));
+    }
 
+    final double textScaleFactor = MediaQuery.of(context, nullOk: true)?.textScaleFactor ?? 1.0;
     return new GestureDetector(
       onTap: onPressed,
       behavior: HitTestBehavior.opaque,
-      child: new Center(
+      child: new Container(
+        alignment: Alignment.center,
+        padding: new EdgeInsets.all(10.0 * textScaleFactor),
```

Comment:
Wondering whether we should have a text size aware proportional edge inset. Would this happen commonly?

Reply:
Yeah, that's probably a good idea.  Here I was just emulating what happens on iOS when the text size increases, and it probably happens other places.